projects/plugins/jetpack/_inc/client/traffic/seo.jsx

```diff
@@ -53,23 +147,108 @@ export const SEO = withModuleSettingsFormHelpers(
 						</p>
 						<ModuleToggle
 							slug="seo-tools"
-							activated={ isActive }
-							toggling={ this.props.isSavingAnyOption( 'seo-tools' ) }
+							activated={ isSeoActive }
+							toggling={ this.props.isSavingAnyOption( seo.module ) }
+							disabled={
+								this.props.isSavingAnyOption( this.constants.moduleOptionsArray ) ||
+								hasConflictingSeoPlugin
+							}
 							toggleModule={ this.props.toggleModuleNow }
 						>
 							{ __( 'Customize your SEO settings', 'jetpack' ) }
 						</ModuleToggle>
 					</SettingsGroup>
-					{ isActive && ! this.props.isOfflineMode && (
-						<Card
-							compact
-							className="jp-settings-card__configure-link"
-							onClick={ this.trackConfigureClick }
-							href={ this.props.configureUrl }
-						>
-							{ __( 'Customize your SEO settings', 'jetpack' ) }
-						</Card>
-					) }
+					{ isSeoActive &&
+						! this.props.isOfflineMode &&
+						! isFetchingPluginsData( this.props.state ) &&
+						! hasConflictingSeoPlugin && (
+							<div>
+								<SettingsGroup>
```

Comment:
I wonder if this one (and the one below) should also be `FoldableCard`, hidden by default, to avoid making the settings page too long?

Reply:
Yes, I think it's a good idea to add them as foldable cards https://github.com/Automattic/jetpack/pull/18810/commits/87d76d81562c2ad6beee49a5328dcd4c88044927